cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -674,6 +674,16 @@
     public static final String SSL_ENABLED = "ssl.enabled";
 
     /**
+     * Authorization.
+     */
+    public static final class Authorization {
+      /** Enables authorization */
+      public static final String ENABLED = "security.authz.enabled";
+      /** AuthorizationHandler class name */
+      public static final String HANDLER_CLASS = "security.authz.handlerClassName";
```

Comment:
Probably need an admin user name who can set the ACLs? Fine if its part of another PR.